pysqldb3/tests/test_backup_table.py

```diff
@@ -278,6 +278,60 @@ def test_messy_table(self):
         if os.path.isfile(test_back_file):
             os.remove(test_back_file)
         assert not os.path.isfile(test_back_file)
+
+    def test_backup_tables_capitalized(self):
+        db.drop_table(table=test_pg_to_backup.capitalize(), schema=pg_schema)
+
+        # table schema
+        db.query(f"""
+            CREATE TABLE {pg_schema}.{test_pg_to_backup.capitalize()} (
```

Comment:
i dont think this will work, if the table name isnt wrapped in `""` or [] for MS the table name will not be capitalized in the database

Reply:
when i made these tests, i wasn't that concerned about the actual letter case for the tables that get read into the database (like whether the table in the db is actually called "table123" or "Table123"). rather, i flagged that if i were a user who used the `backup_table` function and typed "Table123" instead of "table123" as the table name argument, the `backup_table` function in PG wouldn't work and would error out. This only happens in the PG test; MS SQL seems to standardize the name whether you use capitals or not.

maybe this is more of a question of how you want the function to work. Should the backup tables maintain the lettercase for the input (in turn, i should add the "" quotes in), or should the function just standardize the input table name (make everything lowercase regardless of how you enter it)? 

I can easily change the tests to remove the `.capitalize()` suffix from all the entries except the function call (if we should just standardize the name to lowercase). let me know what you think!

Reply:
but if the table in the db is actually named table123 than backing up Table123 should error since that table doesn't exist, just like if i type table321 instead of table123. 

the backup function should preserve the table name it is in the database, so if the name is capitalized in the database it should be in the back, but if not then it should match the formatting of the table. 

right now almost everything in pysqldb3 will standardize to lowercase by default, there are some edge cases that will keep the input format but they are rare. I do want to go though and fix it so everything is lower unless specified that case is important but that hasnt been done yet

Reply:
OK i've altered the PG `test_backup_tables_capitalized` test to change the way the table name is capitalized and the test passes now. Yes, not the biggest issue but I was surprised when the same test passed for one server type but not the other due to capitalizing the table name.